<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBoxPositionalAudioServers</class>
 <widget class="QGroupBox" name="GroupBoxPositionalAudioServers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBoxPositionalAudioServers</string>
  </property>
  <property name="title">
   <string>Server Settings</string>
  </property>
  <widget class="QListWidget" name="listWidget_servers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>211</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxSettings">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>81</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Enabled</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_SendInterval">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Interval</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_SendInterval">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string notr="true"> sec</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unblock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>Unblock</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_SendIntervalSilentInc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string notr="true"> sec</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.200000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_SendIntervalSilentInc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Silent +</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>235</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Add current server tab as custom server setting</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_remove">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>235</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Remove the selected rows custom server setting</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
